src/components/navigation/section-document-or-browser.tsx

```diff
@@ -33,12 +36,16 @@ export const SectionDocumentOrBrowser: React.FC<IProps> = observer(function Sect
   const queryClient = useQueryClient();
   const user = useUserStore();
   const classStore = useClassStore();
+  const [scrollerCollapsed, setScrollerCollapsed] = useState(false);
+  const [showLeftScrollArrow, setShowLeftScrollArrow] = useState(false);
+  const [showRightScrollArrow, setShowRightScrollArrow] = useState(false);
+  const documentScrollerRef = useRef<HTMLDivElement>(null);
   const navTabSpec = appConfigStore.navTabs.getNavTabSpec(tabSpec.tab);
-
   const subTabs = tabSpec.subTabs;
   const tabState = navTabSpec && ui.tabs.get(navTabSpec?.tab);
   const _subTabIndex = subTabs.findIndex((subTab) => tabState?.openSubTab === subTab.label);
   const subTabIndex = _subTabIndex < 0 ? 0 : _subTabIndex;
```

Comment:
```suggestion
  const _subTabIndex = Math.max(subTabs.findIndex((subTab) => tabState?.openSubTab === subTab.label), 0);
```
I can't help but offer these tiny little optimizations 😃